BUILD.gn

```diff
@@ -267,6 +267,10 @@ if (current_toolchain != "${dir_pw_toolchain}/default:default") {
         "//scripts/py_matter_yamltests:matter_yamltests.tests",
         "//src:tests_run",
       ]
+
+      if (current_os == "linux") {
```

Comment:
Please document why this is Linux-only.  Ideally this would run on Darwin as well, so people running tests locally run these tests.

Reply:
I had it like this because I only tested on linux, however I believe technically things should work on mac.

I created a commit to add mac into the accepted list here, to see if things pass. If not I will just document.

Reply:
Darwin CI seems to pass, so both platforms accepted now.